.circleci/config.yml

```diff
@@ -43,7 +44,7 @@ jobs:
               --user "CircleCI Deploy Job <ci-build@circleci.com>" \
               --message \
             "Deploy webpage on `date --utc --iso-8601`
-            
+
```

Comment:
Will deleting this whitespace mess up the de-indenting here?

Reply:
This was done by my editor (Emacs) automatically, but I don't think it will mess up the format.

Reply:
Okay the online YAML parser at https://yaml-online-parser.appspot.com/ seems to parse this the same way with the spaces stripped, so I am okay with merging this. Although in general, it'd be best to only remove whitespace like this when it's a typo and definitely-non-functional.